network/tests/test_ethtool.py

```diff
@@ -453,7 +453,9 @@ def send_ethtool_ioctl_mock(iface, sckt, data):
 
 @pytest.mark.skipif(platform.system() == 'Windows', reason="Only runs on Unix systems")
 @mock.patch('datadog_checks.network.ethtool._send_ethtool_ioctl')
-def test_collect_ena(send_ethtool_ioctl, check):
+@mock.patch('datadog_checks.network.network.Platform.is_bsd', return_value=False)
+@mock.patch('datadog_checks.network.network.Platform.is_linux', return_value=True)
```

Comment:
Would be nice if we could refactor this into a pytest fixture / decorator.

Reply:
That's a rabbithole I don't want to get into right now